src/components/Lookup/styled/selectedInput.js

```diff
@@ -1,34 +1,26 @@
 import styled from 'styled-components';
-import { COLOR_GRAY_4, COLOR_GRAY_3 } from '../../../styles/colors';
+import attachThemeAttrs from '../../../styles/helpers/attachThemeAttr';
 import { FONT_SIZE_TEXT_MEDIUM } from '../../../styles/fontSizes';
 import Input from '../../Input/styled/input';
-import getTheme from '../../../styles/helpers/getTheme';
 
-const StyledSelectedInput = styled(Input).attrs(props => {
-    const brandMainColor = getTheme(props).palette.brand.main;
-
-    return {
-        // TODO: move up to defaultTheme or normalizeTheme
-        brandShadow: `0 0 2px ${brandMainColor}`,
-    };
-})`
+const StyledSelectedInput = attachThemeAttrs(styled(Input))`
     box-shadow: inset 0 0 0 3px #fff, inset 0 0 0 4px #dddbda;
     caret-color: transparent;
-    color: ${COLOR_GRAY_4};
+    color: ${props => props.palette.text.label};
     font-size: ${FONT_SIZE_TEXT_MEDIUM};
     padding-right: 2.35rem;
     text-overflow: ellipsis;
 
     :focus,
     :active {
-        box-shadow: inset 0 0 0 2px #fff, inset 0 0 0 3px #dddbda, ${props => props.brandShadow};
+        box-shadow: inset 0 0 0 2px #fff, inset 0 0 0 3px #dddbda, ${props => props.shadows.brand};
```

Comment:
Here where #fff is used we need to pass mainBackground because this part of this complex shadow need to looks like the background of the input, and where #dddbda is used we need to pass one of the border colors